sdk/testing/src/main/java/io/opentelemetry/sdk/testing/trace/TraceUtil.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.testing.trace;
+
+import io.opentelemetry.sdk.trace.data.SpanData;
+import java.util.Comparator;
+
+public class TraceUtil {
+
+  /** Compare spans by start time, placing parents before their children as a tiebreaker. */
+  public static final Comparator<SpanData> SPAN_DATA_COMPARATOR =
```

Comment:
Why include this here rather than as a private field in `TracesAssert` where its used?

Reply:
to make it usable in libraries - I have such a library already - and want to upstream the changes.

Reply:
made package private